src/assignments/11-higher-order-function.js

```diff
@@ -32,3 +32,12 @@ const users = [
 ];
 
 // Program 1: create a function and print the user details whose age>=30 (use filter and map)
+
+const newArray = () => {
+  const usersAbove30 = users.map(function() {
+      return users.filter(users => users.age > 30)[0]
+  })
+  console.log(usersAbove18)
+} 
+
```

Comment:
the requirement means: you can use map and filter to filter people who has age over 30. 
Use a method for filter and one for map in separate way!